src/components/Gallery/_gallery.scss

```diff
@@ -36,21 +39,38 @@
 }
 
 .cc-gallery__media-item {
-  @include mq($until: sm) {
-    + .cc-gallery__media-item {
-      margin-top: calc(4 * var(--space-unit));
-    }
-  }
-
-  @include mq(sm) {
-    @include flex-grid-item(2);
-  }
+  @extend %flex-grid-item;
+  @include flex-grid-item(2);
+  appearance: none;
+  background: transparent;
+  border: 0;
   
-  @include mq(md) {
+  @include mq(sm) {
     @include flex-grid-item(3);
   }
 }
 
 .cc-gallery__media-content {
   @extend %image-responsive;
 }
+
+.break {
+  flex-basis: 100%;
+  height: 0;
+}
```

Comment:
What does this do?

Reply:
aha! magic (ssh)

alright then, it's a means of forcing flexbox items on to the next row. In this instance I'm using it to ensure the lead image (if there is one) is the only item on its row. Not the most helpfully named class right now I'll admit - how about `u-flexgrid-break`?

Reply:
ah okay - would `flex-wrap: wrap` on the parent work, instead?

Reply:
sadly not - as discussed this does appear to be the neatest solution to the problem

Reply:
agreed